subsys/bluetooth/mesh/shell/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+#
+# Copyright (c) 2022 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+zephyr_library()
+
+zephyr_library_sources_ifdef(CONFIG_BT_MESH_SHELL shell_model_utils.c)
+zephyr_library_sources_ifdef(CONFIG_BT_MESH_SHELL_MODELS_ONOFF_CLI shell_onoff_cli.c)
```

Comment:
This is a big nitpicking, but is there a need in `_models_`/`_model_` suffix in function names/Kconfig options? Or at least some consistency should be here, e.g.: `shell_model_utils.c` (model, singular form), but `CONFIG_BT_MESH_SHELL_MODELS_<MODEL_NAME>` (models, plural form), or `shell_onoff_cli.c` without `_model_` suffix, but why not `shell_utils.c`?

Reply:
Agreed, I will take a look at it. Naming convention was never my strong suit.

Reply:
> Naming convention was never my strong suit.

Same for me 😄